hack/images

```diff
@@ -21,35 +21,35 @@ usage() {
 	exit 1
 }
 
-if [ -z "$TAG" ] || [ -z "$REPO" ]; then
+if [ -z "${TAG}" ] || [ -z "${REPO}" ]; then
 	usage
 fi
 
 imageDockerCommon() {
-	if echo $PLATFORMS | grep , >/dev/null; then
-		echo "PREFER_BUILDCTL=1 needs to be set to build images for $PLATFORMS"
+	if echo ${PLATFORMS} | grep , >/dev/null; then
+		echo "PREFER_BUILDCTL=1 needs to be set to build images for ${PLATFORMS}"
 		exit 1
 	fi
 	set -x
-	docker build $@ --platform=$PLATFORMS -t $REPO:$TAG .
-	docker build $@ --platform=$PLATFORMS -t $REPO:$TAG-rootless --target rootless .
+	docker build $@ --"platform=${PLATFORMS}" -t "${REPO}:${TAG}" .
+	docker build $@ --"platform=${PLATFORMS}" -t "${REPO}:${TAG}-rootless" --target rootless .
```

Comment:
Placement of quotes looks strange to me. I suggest to either quote the whole thing (`"--platform=${PLATFORMS}"`) or just the part that might contain whitespace (`--platform="${PLATFORMS}"`).

Reply:
oh, good catch; I think this might've been a wrong find/replace on my side